aries-site/src/examples/components/card/FeaturePromotionCard.js

```diff
@@ -26,8 +25,7 @@ export const FeaturePromotionCard = () => (
     actions={
       <Button
         label="Launch demo"
-        icon={<FormNextLink />}
-        primary
+        kind="cta-primary"
```

Comment:
Not related to this line, but to this card -- Since this card is using gradient type, we need to have the card's background set to white per brand guidelines. 

"Typography style 3 – texture or 2-3 color gradient in type (T3)
Rules
Use only over a white background.
Primarily use Sentence case, avoid Title Case (apart from in names) and use ALL CAPS as minimally as possible."

Reply:
Good note, I'll update this card.

Reply:
Done.